troposphere/static/js/components/modals/image_version/requirements/EditMinimumRequirementsView.react.js

```diff
@@ -0,0 +1,43 @@
+define(function (require) {
+
+  var React = require('react/addons');
```

Comment:
@prosif - can you change this to the ES6 Module `import` & `export` format? _([notes from tech talk](https://gist.github.com/lenards/711efb16001d02152598))_

So this should be (I believe): 
```javascript
import React from 'react/addons'; 
```

And you'd remove the wrapping `define( ... )`

Plus, change the `return React.createClass({ ... });` to `export default React.createClass({ ... })`.